.github/workflows/release.yml

```diff
@@ -0,0 +1,58 @@
+name: Release
+
+on:
+  push:
+    branches:
+      - master
+    tags: [ v.* ]
+
+jobs:
+  release:
+    # runs on main repo only
+    if: github.repository == 'akka/akka-projection'
+    name: Release
+    # the release environment provides access to secrets required in the release process
+    # https://github.com/akka/akka-projection/settings/environments
+    environment: release
+    runs-on: ubuntu-18.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          # we don't know what commit the last tag was it's safer to get entire repo so previousStableVersion resolves
+          fetch-depth: 0
+
+      - name: Checkout GitHub merge
+        if: github.event.pull_request
+        run: |-
+          git fetch origin pull/${{ github.event.pull_request.number }}/merge:scratch
+          git checkout scratch
+
+      - name: Setup Scala with JDK 8
+        uses: olafurpg/setup-scala@v10
+
+      - name: Cache Coursier cache
+        uses: coursier/cache-action@v5
+
+      - name: Whitesource check policies and update
+        env:
+          WHITESOURCE_PASSWORD: ${{ secrets.WHITESOURCE_PASSWORD }}
+        run: sbt ";whitesourceCheckPolicies ;whitesourceUpdate"
```

Comment:
do we run `whitesource` only on releases?
my understanding is that it should run on `master` (so that, if something pops-up can be corrected before the next release)

Reply:
Yeah, that's what we have been doing. We can revise it. I think you have a good point. Can we open a new ticket to discuss it? Maybe in akka-meta.

Reply:
My travis foo might be outdated but:
https://github.com/akka/akka-projection/blob/8c317094c54f4499bd7fbdd893e13a6d208ef3bf/.travis.yml#L54-L56
is in stage `whitesource` and is running on `master`:
https://github.com/akka/akka-projection/blob/8c317094c54f4499bd7fbdd893e13a6d208ef3bf/.travis.yml#L73-L75
as the Travis UI shows here:
https://travis-ci.com/github/akka/akka-projection/builds/222018831

Reply:
You are right. I think we have just copied this from Alpakka and JDBC plugin without noticing it. I assumed we were only checking it on release. 

I will change and also adapt in the other projects.

Reply:
@andreaTP, I think we got confused here. 

The workflow name is `release`, but it's run in master after a merge.
We don't want it to run on pull requests though. So, that aligns with the previous travis settings, run if on master as is a push (merge) or if it has a tag.